### PR TITLE
Rebase on 1.1.3

### PR DESCRIPTION
**Pull request by mwleeds**
This is the current state of upstream master (which will become 1.1.0) plus 14 downstream commits. The only merge conflicts were minor and are noted in the commit messages.

https://phabricator.endlessm.com/T24078

### CODE REVIEW

**Comment by mwleeds** [comment]
Hmm, looks like the tests are failing on our CI, even though they're succeeding on the upstream one

**Comment by dbnicholson** [comment]
Our tests are based on `make distcheck`, so likely there's something wrong with the dist handling:
```
PATH=$(cd . && pwd):${PATH} tests/make-test-runtime.sh tests/runtime-repo org.test.Platform ""
/bin/bash: tests/make-test-runtime.sh: No such file or directory
```
Seems like that script isn't being distributed. Or maybe that command isn't handling `srcdir != builddir` correctly (which distcheck enforces). You should try running `make distcheck` in your checkout.

File: tests/Makefile.am.inc [hunk 1]
```diff
@@ -109,7 +109,7 @@ tests/test-%.wrap:
 
 tests/runtime-repo: tests/make-test-runtime.sh flatpak
 	rm -rf tests/runtime-repo
-	PATH=$$(cd $(top_builddir) && pwd):$${PATH} tests/make-test-runtime.sh tests/runtime-repo org.test.Platform ""
+	PATH=$$(cd $(top_srcdir)/tests && pwd):$${PATH} make-test-runtime.sh tests/runtime-repo org.test.Platform ""
```
Comment (dbnicholson) [inline tests/Makefile.am.inc]: Yeah, that looks right. I don't even know why the path is being adjusted, though. The script can be called directly with `$(top_srcdir)/tests/make-test-runtime.sh`. Why have the shell find the program when you know exactly where it is?

FWIW, the whole `$$(cd $(top_somedir) && pwd)` seems silly, too. The same thing should be available with `$(abs_top_srcdir)`. But I know that's not what you're hunting here.

**Comment by mwleeds** [comment]
bot, retest this please

**Comment by mwleeds** [comment]
This is now freshly rebased against the 1.1.1 upstream release, and includes every currently on the downstream master.

**Comment by mwleeds** [comment]
This is now rebased again on upstream master (which is a few commits past 1.1.3) and includes the downstream commits on master, with the exception of the last three which are related to translations. I want to talk to @wjt about how to resolve those merge conflicts.

**Comment by mwleeds** [comment]
Per @wjt we're going to leave the translation commits out and he will add them back later. So this is ready for review but can't actually be merged until (a) the OSTree rebase has been merged and (b) the packaging branch here has been updated (I think it will need changes to go from 1.1.1 to 1.1.3)

**Comment by wjt** [comment]
Could you `git cherry-pick 3eb2166a857da34dc174ca901d55b59705055a00` or squash it into the commit which introduced that string, please?

**Pull request merged**